src/parameters.toml

```diff
@@ -1599,6 +1604,26 @@ value = 6
 type = "float"
 description = "Limiter for shape parameter mu for ice in P3 scheme. See eq 3 in Morrison and Milbrandt 2015. Units: [-]"
 
+[water_density]
+value = 1000
+type = "float"
+description = "Density of water in kg/m3"
+
+[freeze_temp]
+value = 273.15
+type = "float"
+description = "Freezing temperature in K"
+
+[p3_ventillation_a]
+value = 0.78
+type = "float" 
+description = "a coefficient for ventillation factor calculations in p3 tendencies" 
```

Comment:
```suggestion
description = "a coefficient for ventillation factor calculations in p3 tendencies, unitless. " 
```

Reply:
Do we have a reference for ventilation coeff values? If not we can put `From Seifert and Beheng, 2006. DOI: 10.1007/s00703-005-0112-4`